### PR TITLE
Svs shop page 2

### PR DESCRIPTION
**Pull request by Sean-Stille**
Created Shop page, merged with release to avoid merge conflicts

### CODE REVIEW

File: AcademicReward/Views/ShopPage.xaml [hunk 1]
```diff
@@ -5,9 +5,42 @@
...
+            
+        <Button ImageSource="shopdemoticket.png"
+               Clicked="Ticket_Clicked" Scale="0.5"
+                   WidthRequest="300" MaximumHeightRequest="200" /> 
+
+
+
+
+        <HorizontalStackLayout>
+
```
Comment (Wil-LaLonde) [inline AcademicReward/Views/ShopPage.xaml:30]: Do you need this HorizontalStackLayout anymore?

File: AcademicReward/AcademicReward.csproj [hunk 2]
```diff
@@ -53,13 +53,21 @@
 		<MauiAsset Include="Resources\Raw\**" LogicalName="%(RecursiveDir)%(Filename)%(Extension)" />
 	</ItemGroup>
 
+	<ItemGroup>
```
Comment (Wil-LaLonde) [inline AcademicReward/AcademicReward.csproj:56]: Do you know what this remove is doing here? Is this adding the images to your list? I'm curious

**Review by Wil-LaLonde — COMMENTED** [review]
Check comments

**Review by Sean-Stille — COMMENTED** [review]
Removed horizontal stack

**Review by Wil-LaLonde — APPROVED** [review]

**Pull request merged**